olive/passes/onnx/gptq.py

```diff
@@ -0,0 +1,149 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+import logging
+from typing import Any, Dict
+
+import torch
+
+from olive.hardware.accelerator import AcceleratorSpec
+from olive.model import PyTorchModelHandler
+from olive.model.handler.onnx import ONNXModelHandler
+from olive.passes import Pass
+from olive.passes.onnx.common import get_external_data_config
+from olive.passes.onnx.gptq_utils import QuantLinearORT
```

Comment:
nit: since this is a util, it is better to be delayed imported inside the `_run_for_config` method to be safe. Doesn't look like an issue here since it is only using torch and its submodules.